server/src/main/java/org/spine3/server/storage/memory/InMemoryStandStorage.java

```diff
@@ -113,12 +115,15 @@ protected EntityStorageRecord readRecord(AggregateStateId id) {
     protected void writeRecord(AggregateStateId id, EntityStorageRecord record) {
         final TypeUrl recordType = TypeUrl.of(record.getState()
                                                     .getTypeUrl());
-        checkState(id.getStateType().equals(recordType), "The typeUrl of the record does not correspond to id");
+        final TypeUrl recordTypeFromId = id.getStateType();
+        checkState(
+                recordTypeFromId.equals(recordType),
+                String.format(TYPE_URL_MISMATCH_MESSAGE_PATTERN,
+                              recordType, recordTypeFromId));
```

Comment:
You can move this line back and concatenate it with the previous line. Just to make the whole expression not so tall.